src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -112,6 +124,25 @@
         :subtitle      network-name
         :subtitle-type :network}]]]))
 
+(defn tabs
+  [_]
+  (let [selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn [{:keys [traits chain-id] :as _props}]
+      [:<>
+       [quo/tabs
+        {:size           32
+         :style          style/tabs
+         :scrollable?    true
+         :default-active @selected-tab
+         :data           tabs-data
+         :on-change      #(reset! selected-tab %)}]
+       (condp = @selected-tab
+         :overview [:<>
+                    [info chain-id]
+                    [traits-section traits]]
+         :activity [activity-section]
+         [rn/view])])))
```

Comment:
Isn't it just the same as just returning `nil`?

Reply:
Why do we have an empty view here? Either we should return something or not 🤔

Reply:
Yeah, I think nil should be good)